config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -4376,6 +4376,11 @@
 # Chat delivery expiration (in seconds).  Default is 300 seconds (5 minutes)
 #chat.delivery.ttl=300
 
+# SAK-40937
+#DEFAULT: 5000
+#chat.pollInterval=7000
```

Comment:
there should be some explanation for these properties! One of the reasons for documenting them!